examples/openshift-compatible/README.md

```diff
@@ -52,22 +54,52 @@ prometheus-node-exporter:
 
 alloy:
   alloy:
-    listenPort: 8080
     securityContext:
+      runAsUser: 473
+      runAsGroup: 473
```

Comment:
I think this needs to come with a huge warning that it would be using undocumented and unsupported functionality (essentially experimental). We don't know anything about the future of the non-root user in the Alloy container, including the user/group IDs.

Reply:
Re-rolled the example without the user assignment.